canal/local.go

```diff
@@ -0,0 +1,108 @@
+package canal
+
+import (
+	"context"
+
+	"github.com/go-mysql-org/go-mysql/mysql"
+	"github.com/go-mysql-org/go-mysql/replication"
+	"github.com/pingcap/errors"
+)
+
+// BinlogFileDownload download the binlog file from cloud computing platform (etc. aliyun)
+type BinlogFileDownload func(mysql.Position) (localBinFilePath string, err error)
```

Comment:
```suggestion
// BinlogFileDownloader downloads the binlog file and return the path to it. It's often used to download binlog backup from RDS service.
type BinlogFileDownloader func(mysql.Position) (localBinFilePath string, err error)
```

and please change other comments

Reply:
I have changed the function name, please review again